test/runtime_tests.cc

```diff
@@ -1013,5 +1015,73 @@ namespace detail {
 		dry_run_with_nodes(nodes);
 	}
 
+	TEST_CASE_METHOD(test_utils::mpi_fixture, "Config reads environment variables correctly", "[env-vars][config]") {
+		std::unordered_map<std::string, std::string> valid_test_env_vars{
+		    {"CELERITY_LOG_LEVEL", "debug"},
+		    {"CELERITY_GRAPH_PRINT_MAX_VERTS", "1"},
+		    {"CELERITY_DEVICES", "0 0"},
```

Comment:
You could make the test a bit more robust by using something other than 0/0 here.